spark-jobs/src/main/scala/za/co/absa/enceladus/common/CommonJobExecution.scala

```diff
@@ -0,0 +1,194 @@
+package za.co.absa.enceladus.common
+
+import java.text.MessageFormat
+import java.time.Instant
+
+import com.typesafe.config.ConfigFactory
+import org.apache.spark.sql.SparkSession
+import org.slf4j.{Logger, LoggerFactory}
+import za.co.absa.atum.AtumImplicits
+import za.co.absa.atum.core.Atum
+import za.co.absa.enceladus.common.plugin.PostProcessingService
+import za.co.absa.enceladus.common.plugin.menas.{MenasPlugin, MenasRunUrl}
+import za.co.absa.enceladus.dao.MenasDAO
+import za.co.absa.enceladus.dao.rest.MenasConnectionStringParser
+import za.co.absa.enceladus.model.Dataset
+import za.co.absa.enceladus.plugins.builtin.errorsender.params.ErrorSenderPluginParams
+import za.co.absa.enceladus.plugins.builtin.errorsender.params.ErrorSenderPluginParams.ErrorSourceId
+import za.co.absa.enceladus.utils.fs.FileSystemVersionUtils
+import za.co.absa.enceladus.utils.general.ProjectMetadataTools
+import za.co.absa.enceladus.utils.performance.PerformanceMeasurer
+import za.co.absa.enceladus.utils.time.TimeZoneNormalizer
+
+import scala.util.control.NonFatal
+import scala.util.{Failure, Success, Try}
+
+trait CommonJobExecution {
+  TimeZoneNormalizer.normalizeJVMTimeZone()
+  protected val log: Logger = LoggerFactory.getLogger(this.getClass)
+  protected val conf = ConfigFactory.load()
+  protected val menasBaseUrls = MenasConnectionStringParser.parse(conf.getString("menas.rest.uri"))
+
+  protected def standardizationPath(jobConfig: JobCmdConfig, reportVersion: Int) =
```

Comment:
as this is a method I would expect some verb like create, generate or something.